okhttp-hpacktests/src/test/java/okhttp3/internal/http2/HpackDecodeTestBase.java

```diff
@@ -23,6 +23,8 @@
 import okhttp3.internal.http2.hpackjson.HpackJsonUtil;
 import okhttp3.internal.http2.hpackjson.Story;
 import okio.Buffer;
+import org.junit.Assume;
```

Comment:
Unused. Any reason you didn't use `assumeTrue`/`assumeFalse`?

Reply:
Absolutely no good reason :)

More that I was in the middle of an if statement, so I wrote some bad code without thought.  I was testing out the fix for missing git submodules.